tests/unit/amo/containers/TestHome.js

```diff
@@ -168,6 +168,11 @@ describe(__filename, () => {
     });
   });
 
+  it('renders a span with comment for IT', () => {
+    const root = render();
+    expect(root.find('[data-comment="For-IT"]')).toHaveLength(1);
```

Comment:
I'd get rid of the `data-` attribute and I'd try to assert `root.html()` to be sure to find the expected HTML comment.

Reply:
@willdurand , I got the `root.html()` test but I am not sure what else to use if not `data-comment`

Reply:
nothing. why do you need this?

Reply:
How to search for element with specific text, either to put class or a data-attribute?

Reply:
You don't. Can't you use `toContain()`?

Reply:
I tried to use this
`  it('renders a span with comment for IT', () => {`
`   const root = render();`
`    expect(root.find('.Home').html()).toContain('Godzilla of browsers');`
`  });`
which should work but it gives me following error.
<img width="561" alt="screen shot 2017-11-10 at 8 20 55 am" src="https://user-images.githubusercontent.com/5318732/32640793-6417acce-c5f0-11e7-8882-e9dc5658bb15.png">